app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        __class__.people.update({self.name: self})
```

Comment:
Instead of using '__class__.people', you can directly use 'Person.people' to access the class attribute.